lib/wanda_web/controllers/executions_controller.ex

```diff
@@ -0,0 +1,30 @@
+defmodule WandaWeb.ExecutionsController do
+  use WandaWeb, :controller
+  use OpenApiSpex.ControllerSpecs
+
+  alias Wanda.Results
+  alias WandaWeb.Schemas.CheckExecutionResponse
+
+  plug OpenApiSpex.Plug.CastAndValidate, json_render_error_v2: true
+
+  operation :list_checks_executions,
+    summary: "List checks executions",
+    parameters: [
+      group_id: [
+        in: :query,
+        description: "Group ID to filter upon",
+        type: :string,
+        example: "00000000-0000-0000-0000-000000000001"
+      ],
+      page: [in: :query, description: "Page", type: :integer, example: 3],
+      items_per_page: [in: :query, description: "Items per page", type: :integer, example: 20]
+    ],
+    responses: [
+      ok: {"Check executions response", "application/json", CheckExecutionResponse}
+    ]
+
+  def list_checks_executions(conn, params) do
+    results = Results.list_execution_results(params)
+    json(conn, %{data: results})
```

Comment:
Cannot we send simply the results? `json(conn, results)`

Reply:
I wanted to wrap this into a map to make our life easier when we switch to views (Phoenix' DTOs)